events/source/EventQueue.cpp

```diff
@@ -24,9 +24,9 @@ namespace events {
 EventQueue::EventQueue(unsigned event_size, unsigned char *event_pointer)
 {
     if (event_size == 0) {
-        // As static queue (EventQueue(0)) won't perform any access to its dummy buffer
-        // set 1B dummy buffer as pointer to itself
-        equeue_create_inplace(&_equeue, 1, this);
+        // As static queue (EventQueue(0)) won't perform any access to its data buffer
+        // we can pass (0, NULL)
+        equeue_create_inplace(&_equeue, 0, NULL);
```

Comment:
In general should be shifting to `nullptr` in C code, but doesn't matter when calling C code anyway.